venza.py

```diff
@@ -28,68 +30,79 @@ def __str__(self):
         return self.fmt("{name} - {place}")
 
     def __repr__(self):
-        return self.fmt("<Meal('{name}', place='{place}')>")
+        return self.fmt("<Meal('{name}',"
+                        " place='{place}'"
+                        " price={price}"                        
+                        ")>")
 
 
 def horna(day=None, month=None, year=None):
-    if year is None or not year.isdigit():
+    if year is None  or not isinstance(year, int):
         year = int(time.strftime("%Y"))
-    if day is None or not day.isdigit():
+    if day is None  or not isinstance(day, int):
         day = int(time.strftime("%d"))
-    if month is None or not month.isdigit():
+    if month is None  or not isinstance(month, int):
         month = int(time.strftime("%m"))
-
-    lin = HS_URL.format(day, month, year)
-
+    
+    lin = EAM_URL.format(year, month, day)
     req = requests.get(lin)
-
     soup = BeautifulSoup(req.text)
-    tables = soup.find_all('table')
-    hornasoup = BeautifulSoup(str(tables[1]))
+    divs = soup.find_all('div')
     list = []
-    trs = hornasoup.find_all('tr')
-
-    for i in range(1, len(trs)):
-        td = str(trs[i])
-        tdsoup = BeautifulSoup(td)
-        bordel = tdsoup.find_all('td')
-        match = re.findall(r'<td>(.*?)<span', str(bordel[1]), re.DOTALL)
-        bettermatch = re.findall(r'\n\s\s\s\s\s\s\s\s(.*?)\n', match[0])
-        list.append(Meal(bettermatch[0], 'horna'))
+    
+    try:
+        hornasoup = BeautifulSoup(str(divs[18]))
+        pol = re.findall(r'<span class="dish-name">(.*?)</span>', str(hornasoup))
+        prices = re.findall(r'<span class="dish-price">...(.*?)/', str(hornasoup))
+        list =[]
+        for i in range (0,len(pol)):
+            list.append(Meal(pol[i],'horna',prices[i],0))
```

Comment:
That 0 at the end is anything but readable. Can you please use the enumerated value that is defined in this file? It will still be represented as 0 internally.